react-native/react-navigation/unmount-compoenent-on-page-change.mo.md

```diff
@@ -0,0 +1,82 @@
+# [MO] Unmount a component on page change *(~14 min)*
+
+## Owner: Nicolas Djambazian
+
+## Motivation
+
+React-navigation doesn't unmount your page when you go on the next one. That allow fast transition when you go back.
+
+But if you are using a Camera component, a WebView or every compoenent which use a lot of resources and can have side effect if it continue to run background, you need to unmount them when you leave the page.
+
+For that, you can use the [`getPathAndParamsForState` of a Router](https://reactnavigation.org/docs/routers/api).
+
+## Prerequisites *(~5min)*
+ - The page have a container.
+  -You must have a main stack navigation exported in `src/Routing.js`
+
+
+## Steps *(~5min)*
+
+ - Add a props `myFeatureIsActivated` on the component with default value at `true`
+ - In the render method, display you compoenent only if `myFeatureIsActivated` is at true : 
+
+ ```jsx
+return ( 
+  <View>
+    {this.props.cameraIsActivated && <Camera />}
+  </View>
+);
+
+ ```
+
+- Test is your component still work (nothing should have change)
```

Comment:
Can you format the checks like this https://github.com/bamlab/dev-standards/blame/master/backend/graphql-js/getting-started-with-apollo-server-dataloader-knex.mo.md#L129 ?